spec/index.bs

```diff
@@ -9860,10 +9873,10 @@ command encoder can no longer be used.
 
 <script type=idl>
 interface mixin GPUBindingCommandsMixin {
-    undefined setBindGroup(GPUIndex32 index, GPUBindGroup bindGroup,
+    undefined setBindGroup(GPUIndex32 index, GPUBindGroup? bindGroup,
         optional sequence<GPUBufferDynamicOffset> dynamicOffsets = []);
 
-    undefined setBindGroup(GPUIndex32 index, GPUBindGroup bindGroup,
+    undefined setBindGroup(GPUIndex32 index, GPUBindGroup? bindGroup,
```

Comment:
Does this overload need the bind group to be nullable? I remember we talked about how to interpret these values earlier, and I forget where we landed, but it seems like we only really need one variant of the function (the one with the default value for dynamic offsets) to accept null here.

Reply:
Strictly speaking we only need one, but it makes just as much sense in both: both can specify zero dynamicOffsets. So for example WASM bindings could exclusively use the Uint32Array overload instead of having to call the other when `bindGroup` is null.

Reply:
Thanks for keeping an eye out for Wasm. This indeed makes it easier to implement.

Do WebGPU implementations care here if user calls `encoder.setBindGroup(i, undefined, ...);` versus `encoder.setBindGroup(i, null, ...);` ? I find that it would be simpler from marshalling perspective to call `encoder.setBindGroup(i, undefined, ...);` to unbind a bind group, but is that guaranteed to work all the same as passing `null`?

Reply:
It's guarantee to accept both `null` and `undefined`. Per WebIDL spec for [conversions from ECMAScript values into nullable types](https://webidl.spec.whatwg.org/#es-nullable-type):
> 2\. Otherwise, if V is undefined, and T includes undefined, return the unique undefined value.
> 3\. Otherwise, if V is null or undefined, then return the IDL nullable type T? value null.

`T` here is `GPUBindGroup`, which doesn't include `undefined`.